example/web_socket/echo/index.html

```diff
@@ -67,6 +74,8 @@ <h1>Echo</h1>
     <pre id="logs"></pre>
 </div>
 
+<!--<script src="packages/sockjs_client_wrapper/sockjs.js"></script>-->
```

Comment:
should these be committed as being commented out?

Reply:
Yeah I don't have a great way to dynamically add/remove the script via the example UI, so I figured this was a good middle ground for now.